pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -311,6 +311,8 @@ const (
 	SeedExtensionsReady ConditionType = "ExtensionsReady"
 	// SeedGardenletReady is a constant for a condition type indicating that the Gardenlet is ready.
 	SeedGardenletReady ConditionType = "GardenletReady"
+	// SeedBackupBucketReady is a constant for a condition type indicating that the associated BackupBucket is ready.
+	SeedBackupBucketReady ConditionType = "BackupBucketReady"
```

Comment:
I am not completely sure, but it looks like these constants are alphabetically sorted. Could move this one to the top.